### DISCUSSION TITLE
Batch Download

### DISCUSSION THREAD

**Discussion by Sirmobin2001** [amir1376/ab-download-manager#66] [Ideas]
سلام عزیزم
دمت گرم با این برنامه خفنت
یه قابلیت خوبی که میتونی اضافه کنی قابلیت batch هست 
مثلا من یه لینک از 20 تا پارت یه گیم دارم با حذف عدد آخر پارت خود دانلود منیجر بر اساس اون رنج عدد که بهش میدی همه رو تو لیست دانلود اضافه میکنه

**Comment by amir1376** [comment]
@Sirmobin2001 سلام و درود.
پیشنهاد خیلی خوبیه. تو آینده حتما اضافش میکنم
